### DISCUSSION TITLE
iPad menu is shown sliding left to right

### DISCUSSION THREAD

**Discussion by Sharma-Arcadix** [SafeExamBrowser/seb-mac#358] [General]
Hi,
      Even I have disabled the task bar in user interface, the menu is still showing with navigation buttons, which i don't want to be shown.
  (Though option to close windows should be there)
 Please suggest a workaround.

![SEBiOS](https://github.com/SafeExamBrowser/seb-mac/assets/136628899/1bbe49f7-22de-4551-a7b6-54a8648c1a8f)

**Comment by danschlet** [comment]
Please don't create both an issue and a discussion!
